internal/pkg/cli/svc_logs.go

```diff
@@ -244,5 +250,6 @@ func BuildSvcLogsCmd() *cobra.Command {
 	cmd.Flags().BoolVar(&vars.follow, followFlag, false, followFlagDescription)
 	cmd.Flags().DurationVar(&vars.since, sinceFlag, 0, sinceFlagDescription)
 	cmd.Flags().IntVar(&vars.limit, limitFlag, 10, limitFlagDescription)
+	cmd.Flags().StringSliceVar(&vars.taskIDs, tasksFlag, []string{}, tasksLogsFlagDescription)
```

Comment:
do we want this to be a slice of 0 items? or be nil? so that it queries all the streams. I don't know if there is a difference?

Reply:
they don't have any difference since both []string{} and nil has 0 length but for consistency i'll change it to nil!